spec/index.html

```diff
@@ -7882,16 +7872,23 @@ <h2>Definition of SPARQL</h2>
         abstract query is then evaluated on an RDF dataset.</p>
       <section id="initDefinitions">
         <h3>Initial Definitions</h3>
-        <section id="activeGraph">
-          <h4>Active Graph</h4>
+        <section id="sparqlDataset">
+          <h4>RDF Dataset</h4>
+          <div>
+            <p>The concept of an <a data-cite="RDF12-CONCEPTS#dfn-rdf-dataset">RDF Dataset</a> is defined in [[RDF12-CONCEPTS]].</p>
+            <p>For the following definitions, we capture each RDF dataset as a set:<br>
+              { G, (&lt;u<sub>1</sub>&gt;, G<sub>1</sub>), (&lt;u<sub>2</sub>&gt;, G<sub>2</sub>), . .
+              . (&lt;u<sub>n</sub>&gt;, G<sub>n</sub>) }<br>
```

Comment:
An ellipsis does not include spaces between the dots; it's just three-dots. In sample data like this, such an ellipse can arguably be placed at the start of the second line (as I have), or at the end of the first line, or both (though one or the other would be more common).
```suggestion
              { G, (&lt;u<sub>1</sub>&gt;, G<sub>1</sub>), (&lt;u<sub>2</sub>&gt;, G<sub>2</sub>),
              ... (&lt;u<sub>n</sub>&gt;, G<sub>n</sub>) }<br>
```